tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -59,6 +59,7 @@ import (
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	"k8s.io/kubernetes/test/utils"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"github.com/cockroachdb/cockroach/pkg/acceptance/localcluster/tc"
```

Comment:
@yylt I guess it's a mistake made by your editor ;-)

Reply:
It is strange, I don't find this in my commits .